java-frontend/src/main/java/org/sonar/java/se/ProgramState.java

```diff
@@ -293,21 +319,28 @@ public void accept(Symbol symbol, SymbolicValue symbolicValue) {
   }
 
   public ProgramState cleanupConstraints(Collection<SymbolicValue> protectedSymbolicValues) {
-    class CleanAction implements BiConsumer<SymbolicValue, Constraint> {
+    class CleanAction implements BiConsumer<SymbolicValue, PMap<Class<? extends Constraint>, Constraint>> {
       boolean newProgramState = false;
-      PMap<SymbolicValue, Constraint> newConstraints = constraints;
+      PMap<SymbolicValue, PMap<Class<? extends Constraint>, Constraint>> newConstraints = constraints;
       PMap<SymbolicValue, Integer> newReferences = references;
 
       @Override
-      public void accept(SymbolicValue symbolicValue, Constraint constraint) {
+      public void accept(SymbolicValue symbolicValue, PMap<Class<? extends Constraint>, Constraint> constraintPMap) {
+          constraintPMap.forEach((domain, constraint) -> {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)